rskj-core/src/test/java/co/rsk/blockchain/utils/BlockGenerator.java

```diff
@@ -75,7 +77,7 @@ public static BlockGenerator getInstance() {
         return INSTANCE;
     }
 
-
+    private final DifficultyCalculator difficultyCalculator = new DifficultyCalculator(RskSystemProperties.CONFIG);
```

Comment:
Can this field be passed in the constructor? Also changing the deprecated *INSTANCE* to
```java
 private static final BlockGenerator INSTANCE = new BlockGenerator(new DifficultyCalculator(RskSystemProperties.CONFIG));
```

Reply:
This is test code and that change would introduce more changes than I want. Let's leave this to the person in the future that needs to inject a different DifficultyCalculator instance.